AUTHORS

```diff
@@ -0,0 +1,10 @@
+Godot Engine is a project initiated by Juan "reduz" Linietsky and Ariel "punto" Manzur.
```

Comment:
I think this should probably be added in another PR, with a first attempt at writing the mentioned exhaustive list of contributors.